sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/online_deployment.py

```diff
@@ -314,37 +319,27 @@ def _merge_with(self, other: "OnlineDeployment") -> None:
             self.instance_count = other.instance_count or self.instance_count
             self.instance_type = other.instance_type or self.instance_type
 
-    def _filter_datastore_to_rest_object(self):
-        # temporarily storing the data collector in the properties since it is not part of the contract
-        # will be removed once the contract is fixed to reflect data collector attribute
-        if is_private_preview_enabled() and self.data_collector:
-            non_flat_data = {}
-            non_flat_data["data_collector"] = self.data_collector._to_dict()
-            flat_data = flatten(non_flat_data, ".")
-            for k, v in flat_data.items():
-                self.tags[k] = v
-
-    @classmethod
-    def _filter_datastore_from_rest_object(
-        cls, entity: "OnlineDeployment", deployment: RestOnlineDeploymentDetails
-    ) -> "OnlineDeployment":
-        # Data collector is private preview. If Private Preview environment variable is not enable
-        # data collector will be removed from tags. Data Collector values will be stored in tags
-        # until data collector is added to the contract.
-        if not is_private_preview_enabled():
-            del_key = []
-            for k in entity.tags:
-                if k.startswith("data_collector"):
-                    del_key.append(k)
-            if len(del_key) > 0:
-                for k in del_key:
-                    del entity.tags[k]
-        else:
-            unflat_data = unflatten(entity.tags, ".")
-            if unflat_data.get("data_collector", None):
-                entity.data_collector = unflat_data.get("data_collector")
-        entity._provisioning_state = deployment.provisioning_state
-        return entity
+    # @classmethod
```

Comment:
is there a reason why we're keeping this method commented out instead of deleting?